### PR TITLE
bugfix: wrong invocation of SchemaParseException

### PR DESCRIPTION
**Pull request by sebastianneubauer**
seems to be a typo. Produces the error if parsing of a avro schema file fails: 

    except schema.AvroException.SchemaParseException as e:
    AttributeError: type object 'AvroException' has no attribute 'SchemaParseException'

Correct name is most probaply `schema.SchemaParseException`

### CODE REVIEW

File: confluent_kafka/avro/load.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def loads(schema_str):
             return schema.parse(schema_str)
         else:
             return schema.Parse(schema_str)
-    except schema.AvroException.SchemaParseException as e:
+    except schema.SchemaParseException as e:
```
Comment (rnpridgeon) [inline confluent_kafka/avro/load.py]: Do you mind adding a test with an invalid schema, such as the one reported in the original issue.

**Comment by rnpridgeon** [comment]
fixes #376

**Comment by sebastianneubauer** [comment]
[clabot:check]

File: confluent_kafka/avro/load.py (hunk 1 above)
Comment (sebastianneubauer) [reply confluent_kafka/avro/load.py]: Test is added and succeeds locally

Comment (sebastianneubauer) [reply confluent_kafka/avro/load.py]: ah, flake8...hold my beer!

Comment (sebastianneubauer) [reply confluent_kafka/avro/load.py]: fixed flake8 test and added flake8 to the dev requirements

**Comment by rnpridgeon** [comment]
LGTM!

**Pull request merged**